app/src/main/java/org/greenstand/android/TreeTracker/models/messages/MessagesRepo.kt

```diff
@@ -111,6 +112,14 @@ class MessagesRepo(
         return convertMessageEntityToMessage(messagesDao.getMessage(id)!!) as SurveyMessage
     }
 
+    suspend fun checkForUnreadMessages(): Boolean{
+        return messagesDao.getTotalUnreadMessagesCount() >= 1
+    }
+
+    suspend fun checkForUnreadMessagesPerUser(wallet: String): Boolean{
```

Comment:
This returns a boolean. Let's name it something like "doesWalletHaveUnreadMessages(wallet)"

Reply:
Don't forget to make this change.